src/Helpers/Directory/Directory.php

```diff
@@ -46,6 +46,11 @@ public function getPathWithoutBasename(): string
         return Str::substr($this->path(), 0, -strlen($this->getPathBasename()));
     }
 
+    public function getPath(): string
```

Comment:
I'm going to remove this as it's the same thing as getPathBasename, and I feel like getPath could refer to the whole path. I'll add a 'getSiteDirectory' method which is hopefully clearer!

Reply:
Just double checked and it's definitely the same, this line is exactly the same as `basename($this->path)`!